### PR TITLE
fix(mailviewer): Store mail pane height as percentage (not pixels)

### PR DESCRIPTION
**Pull request by castaway**
Doesnit quiiiite work, @tadzik any ideas? (it's not finding the heightOffsetToPercentage function)

### CODE REVIEW

File: src/app/mailviewer/singlemailviewer.component.ts [hunk 1]
```diff
@@ -164,19 +166,27 @@ export class SingleMailViewerComponent implements OnInit, DoCheck, AfterViewInit
...
-    this.resizerQuery.changes.subscribe((resizer: HorizResizerDirective) => {
+    this.resizerQuery.changes.subscribe((resizer: QueryList<HorizResizerDirective>) => {
       setTimeout(() => {
         if (this.adjustableHeight) {
           if (this.previousHeight) {
-            this.resizer.resizePixels(this.previousHeight);
+            // upgrade to Percentage
+            this.previousHeightPercentage = resizer.first.heightOffsetToPercentage(this.previousHeight);
+            this.previousHeight = undefined;
+            localStorage.setItem(resizerHeightKey, '');
```
Comment (tadzik) [inline src/app/mailviewer/singlemailviewer.component.ts:186]: Any reason not to go all-in with `removeItem()`?

File: src/app/mailviewer/singlemailviewer.component.ts [hunk 2]
```diff
@@ -569,11 +579,11 @@ export class SingleMailViewerComponent implements OnInit, DoCheck, AfterViewInit
 
   }
 
-  heightChanged(height: number) {
-    this.height = height;
-    if (height > 0) {
-      this.previousHeight = height;
-      localStorage.setItem(resizerHeightKey, height.toString());
+  heightChanged(percentage: number) {
+    // this.height = height;
```
Comment (tadzik) [inline src/app/mailviewer/singlemailviewer.component.ts:583]: Leftover comment :)

**Review by tadzik — CHANGES_REQUESTED** [review]
Nitpicks; looks good :)

File: src/app/mailviewer/singlemailviewer.component.ts (hunk 1 above)
Comment (castaway) [reply src/app/mailviewer/singlemailviewer.component.ts:186]: Not other than laziness in me looking up what that was called ;)

**Review by tadzik — APPROVED** [review]

**Pull request merged**